### PR TITLE
mate: update to 1.20.0

### PR DESCRIPTION
**Pull request by Sir-Boops**
* Tested on x86_64 appears to work fine
* Compiled on aarch64-musl no errors
* Removed mate-polkit-devel as it dosn't seem to exist anymore(?)

### CODE REVIEW

**Comment by maxice8** [comment]
you're back!

File: srcpkgs/mate-control-center/template [hunk 1]
```diff
@@ -15,7 +15,7 @@ maintainer="Juan RP <xtraeme@voidlinux.eu>"
 license="GPL-2.0-or-later"
```
Comment (maxice8) [inline srcpkgs/mate-control-center/template]: Please change license= on the other packages to match SPDX like this one.

Comment (Sir-Boops) [reply srcpkgs/mate-control-center/template]: So pretty much just add -"or-later" to all of them?

**Comment by maxice8** [comment]
On Wed, Apr 25, 2018 at 05:37:09AM -0700, Sir-Boops wrote:
> @Sir-Boops commented on this pull request.
> 
> ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━
> 
> In srcpkgs/mate-control-center/template:
> 
> > @@ -15,7 +15,7 @@ maintainer="Juan RP <xtraeme@voidlinux.eu>"
>  license="GPL-2.0-or-later"
> 
> So pretty much just add -"or-later" to all of them?
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.*
> 

no, follow SPDX identifiers https://spdx.org/licenses/ , check the source code headers if it's to use -or-later, -only for GPL, LGPL licenses.

**Comment by Sir-Boops** [comment]
Ah will do :ok_hand:

**Comment by Sir-Boops** [comment]
There licenses have been updated! :smiley:

**Pull request merged**